nengo_bones/templates/remote-script.sh.template

```diff
@@ -0,0 +1,74 @@
+{% extends "remote.sh.template" %}
+
+{% block before_script %}
+{{ super() }}
+    {% if azure_name %}
+    if ssh {{ host }} -q exit; then
+        echo "VM already running"
+    else
+        echo "Starting VM"
+        curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+        exe chmod 400 ~/.ssh/azure.pem
+        exe az login --service-principal -u http://travis-ci -p ~/.ssh/azure.pem --tenant appliedbrainresearch.com
+        exe az vm start --resource-group {{ azure_group }} --name {{ azure_name }}
+    fi
+    {% endif %}
+
+    ssh {{ host }} -q "echo 'Connected to {{ host }}'"
+{% endblock %}
+
+{% block remote_install %}
+{{ super() }}
+        # copy environment variables into remote environment
+        export TRAVIS_JOB_NUMBER="$TRAVIS_JOB_NUMBER"
+        export TRAVIS_BRANCH="$TRAVIS_BRANCH"
+        export TRAVIS_TAG="$TRAVIS_TAG"
+        export TEST_ARGS="$TEST_ARGS"
+        export PIP_USE_FEATURE="$PIP_USE_FEATURE"
+        {% if remote_vars %}
+        {% for var, val in remote_vars.items() %}
+        export {{ var }}="{{ val }}"
+        {% endfor %}
+        {% endif %}
+
+        # generate CI scripts on remote device
+        pip install git+https://github.com/nengo/nengo-bones#egg=nengo-bones
+        bones-generate --output-dir .ci ci-scripts || REMOTE_STATUS=1
+
+        {% for line in remote_setup %}
+        {{ line }}
+        {% endfor %}
+
+        bash .ci/{{ remote_script }}.sh install || REMOTE_STATUS=1
+{% endblock %}
+
+{% block remote_script %}
+{{ super() }}
+        export TRAVIS_JOB_NUMBER="$TRAVIS_JOB_NUMBER"
+        export TRAVIS_BRANCH="$TRAVIS_BRANCH"
+        export TRAVIS_TAG="$TRAVIS_TAG"
+        export TEST_ARGS="$TEST_ARGS --cov-report=xml"
+        export PIP_USE_FEATURE="$PIP_USE_FEATURE"
+        {% for var in remote_vars %}
+        export {{ var }}="${{ var }}"
+        {% endfor %}
+
+        echo "Waiting for lock on device $DEVICE_ID"
+        (
+            flock -x -w 540 200 || exit 1
+            bash .ci/{{ remote_script }}.sh script || exit 1
+        ) 200>/var/lock/.travis-ci.exclusivelock."$DEVICE_ID" || REMOTE_STATUS=1
+{% endblock %}
+
+{% block after_script %}
+    {% if remote_script == "docs" %}
```

Comment:
Just a personal preference thing, but I would un-indent these by two spaces just to make it easier to distinguish the templating from the actual code in this block.

Reply:
I'm pretty indifferent (the Jinja templates don't have a well-specified style) but I'm going to leave it as is to be consistent with the other templates in this repo.